pyocd/probe/pydapaccess/interface/pyusb_backend.py

```diff
@@ -91,10 +91,12 @@ def open(self):
         self.kernel_driver_was_attached = False
         try:
             if dev.is_kernel_driver_active(interface_number):
+                LOG.debug("Detaching Kernel Driver of Interface %d from USB device (VID=%04x PID=%04x).", interface_number, dev.idVendor, dev.idProduct)
                 dev.detach_kernel_driver(interface_number)
                 self.kernel_driver_was_attached = True
-        except NotImplementedError as e:
+        except:
```

Comment:
Can we make the exception handler catch all the errors that could be raised by pyusb/libusb without making it a catchall handler? Reason being that pylint and other tools will flag that as problematic code. It also will not allow an `InterruptException` (Ctrl-C) to pass.

Reply:
Is `except (NotImplementedError,usb.core.USBError) as e` okay? :-)